packages/kit/src/core/config/options.js

```diff
@@ -252,7 +252,7 @@ const options = object(
 			),
 
 			// TODO remove this for 1.0
-			target: error((keypath) => `${keypath} is no longer required, and should be removed`),
+			target: warn((keypath) => `${keypath} is no longer required, and should be removed`),
```

Comment:
This seems unrelated to these changes (did you push to the wrong branch?) but, as I mentioned over in #3769, I don't think we should change this anyway.